conf/config.py

```diff
@@ -42,6 +42,8 @@ class BaseConfiguration(object):
     # Available log levels are: debug, info, warn, error.
     LOG_LEVEL = 'info'
 
+    CONSOLE_LOG = False
```

Comment:
This is the config file used in testing or developer mode (https://github.com/redhat-exd-rebuilds/freshmaker/blob/807bf9e3b1bbcd79ce00be615e3a6e9c742e1e17/freshmaker/config.py#L67)

You have to update this in https://github.com/redhat-exd-rebuilds/freshmaker/blob/807bf9e3b1bbcd79ce00be615e3a6e9c742e1e17/freshmaker/config.py#L135

Reply:
After your last explanation, these changes became unnecessary and I just reverted them.